src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -77,47 +77,67 @@ export default class rd2EntryForm extends LightningElement {
     * @description Dynamic render edit form CSS to show/hide the edit form 
     */
     get cssEditForm() {
-        return (!this.isLoading && this.isSettingReady && this.isRecordReady)
+        return (!this.isLoading && this.isSettingReady && this.fieldInfoReady && this.isRecordReady)
             ? ''
             : 'slds-hide';
     }
 
     /*******************************************************************************
-    * @description Get Recurring Donation SObject API name 
+    * @description Set today's date as default Date of Month in New entry form
     */
-    get recurringDonationName() {
-        return getNestedProperty(RECURRING_DONATION_INFO, 'objectApiName');
+    get defaultDayOfMonth() {
```

Comment:
- Let's group all functions for the Day of Month close to each other, ie move wired picklist function and `getTodayDateOfMonth` below this one.
- Should `this.recordId == null` be `this.recordId === null` everywhere it is used? Or let's better set a new var `isNew` that will be set in the `connectedCallback()` if the new RD is to be created and use `isNew` throughout.